src/BubbleSorter.java

```diff
@@ -0,0 +1,32 @@
+public class BubbleSorter {
+
+    @SuppressWarnings("checkstyle:WhitespaceAfter")
+    public void sortTable(int[] table) {
+        int swap;
+        int change = 0;
+        int counter = 0;
+        for (int i = 0; i < table.length; i++) {
+            for (int j = 0; j < table.length - 1; j++) {
+                counter++;
+                if (table[j] > table[j + 1]) {
+                    swap = table[j + 1];
+                    table[j + 1] = table[j];
+                    table[j] = swap;
+                    change++;
+                }
+            }
+            if (change == 0) {
+                break;
+            }
+        }
+
+        System.out.println();
+        System.out.println("Tablica po sortowaniu");
+        for (int j : table) {
+            System.out.printf("%d ", j);
+        }
+        System.out.printf("\nLiczba zmian: %d", change);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.